opts.go

```diff
@@ -178,6 +226,10 @@ func makeOptions(opts []Option) *options {
 	for _, opt := range opts {
 		opt(&o)
 	}
+	if o.fieldValueHandling != nil {
+		o.fieldValueHandlingConfig = New()
+		o.fieldValueHandlingConfig.Merge(o.fieldValueHandling, PathSep(o.pathSep))
+	}
```

Comment:
It's a little confusing to have fieldValueHandling itself just act as a temporary variable. There is no real difference between creating a big map and merging once, or merging multiple times. If performance is the goal, then I would use *Config to build the tree anyways :)

Reply:
Good point!